doc/src/manual/faq.md

```diff
@@ -29,7 +29,7 @@ you can redefine types and constants.  You can't import the type names into `Mai
 to be able to redefine them there, but you can use the module name to resolve the scope.  In other
 words, while developing you might use a workflow something like this:
 
-```
+```julia
```

Comment:
That's a good idea. Regarding this PR, is there anything that should be changed, added, removed, etc, or is it okay as-is?